src/main/java/edu/kis/powp/jobs2d/features/AppFeature.java

```diff
@@ -0,0 +1,20 @@
+package edu.kis.powp.jobs2d.features;
+
+import edu.kis.powp.appbase.Application;
+import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.drivers.SelectDriverMenuOptionListener;
+
+public class AppFeature {
+    private static Application app;
+
+    public static void setupFeaturesMenu(Application application) {
+        app = application;
+        app.addComponentMenu(AppFeature.class, "Features");
+    }
+
+    public static void addFeature(String name, Job2dDriver driver) {
+        SelectDriverMenuOptionListener listener = new SelectDriverMenuOptionListener(driver, DriverFeature.getDriverManager());
```

Comment:
I cannot see the difference in handling addDriver & andFeature. You use the same listener, so the logic seems the same.

Reply:
Those two methods build two different sections of our application menu, that's why we cannot use the same class or methods for both.

Reply:
You should not use the "same class". The behaviour of features and drivers should not be the same ( Now it is defined in one class SelectDriverMenuOptionListener ).